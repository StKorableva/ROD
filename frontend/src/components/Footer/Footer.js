import './Footer.css';
import twitterIcon from '../../assets/img/twitter.svg';
import instagramIcon from '../../assets/img/instagram.svg';
import facebookIcon from '../../assets/img/facebook.svg';

function Footer(props) {
	function getCurrentYear() {
		return new Date().getFullYear();
	}
	return (
		<div className="footer">
			<div className="footer__textBox">
				<p className="footer__textBox__heading">ROD</p>
				<p className="footer__textBox__copyright">
					Â© {getCurrentYear()} Some amazing team
				</p>
			</div>
			<div className="footer__socials">
				<img
					className="footer__socials__twitter"
					src={twitterIcon}
					alt="houston we have a problems"
				></img>
				<img
					className="footer__socials__facebook"
					src={facebookIcon}
					alt="houston we have a problems"
				></img>
				<img
					className="footer__socials__instagram"
					src={instagramIcon}
					alt="houston we have a problems"
				></img>
			</div>
		</div>
	);
}

export default Footer;
