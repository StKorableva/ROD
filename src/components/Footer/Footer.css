.footer {
	background-color: black;
	color:white;
	display: flex;
	flex-direction: row;
	justify-content: space-around;
}
.footer__textBox{
	display: flex;
	flex-direction: column;
	justify-content: space-around;
}
.footer__socials{
	display: flex;
	flex-direction: row;
	justify-content: space-around;
}
.footer__socials img{
	margin: 10px;
}