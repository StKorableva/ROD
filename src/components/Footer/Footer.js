import './Footer.css';
import twitterIcon from '../../assets/img/twitter.svg';
import instagramIcon from '../../assets/img/instagram.svg';
import facebookIcon from '../../assets/img/facebook.svg';

function Footer(props) {
	function getCurrentYear() {
		return new Date().getFullYear();
	}
	return (
		<div className="footer">
			<div className="footer__textBox">
				<p className="footer__textBox__heading">ROD</p>
				<p className="footer__textBox__copyright">Â© {getCurrentYear()} Some amazing team</p>
			</div>
			<div className="footer__socials">
				<img className="footer__socials__twitter" src={twitterIcon} alt=""></img>
				<img className="footer__socials__facebook" src={facebookIcon} alt=""></img>
				<img className="footer__socials__instagram" src={instagramIcon} alt=""></img>
			</div>
		</div>
	);
}

export default Footer;
