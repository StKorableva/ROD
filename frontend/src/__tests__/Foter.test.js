import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Footer from '../components/Footer/Footer';

describe('base test', () => {
	let component = shallow(<Footer></Footer>);

	it('render test', () => {
		expect(component).toHaveLength(1);
	});

	it('snapshot test', () => {
		expect(toJSON(component)).toMatchSnapshot();
	});
});
