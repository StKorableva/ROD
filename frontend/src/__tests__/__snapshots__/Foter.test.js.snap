// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`base test snapshot test 1`] = `
<div
  className="footer"
>
  <div
    className="footer__textBox"
  >
    <p
      className="footer__textBox__heading"
    >
      ROD
    </p>
    <p
      className="footer__textBox__copyright"
    >
      Â© 
      2021
       Some amazing team
    </p>
  </div>
  <div
    className="footer__socials"
  >
    <img
      alt=""
      className="footer__socials__twitter"
      src="twitter.svg"
    />
    <img
      alt=""
      className="footer__socials__facebook"
      src="facebook.svg"
    />
    <img
      alt=""
      className="footer__socials__instagram"
      src="instagram.svg"
    />
  </div>
</div>
`;
